src/test/java/org/broadinstitute/consent/http/service/DarCollectionServiceTest.java

```diff
@@ -238,7 +238,7 @@ void testCancelDarCollectionAsAdmin() {
   @Test
   void testCancelDarCollectionAsChair_ChairHasDatasets() {
     User user = new User();
-    user.setEmail("email");
+    user.setUserId(RandomUtils.nextInt(1, 10));
```

Comment:
why a random user id for testing?

Reply:
> why a random user id for testing?

The only reason is to have something to query on. The actual value is irrelevant.